fenix/config/detekt.yml

```diff
@@ -726,22 +789,3 @@ style:
     active: true
     excludeImports:
       - 'java.util.*'
-
-# Custom Rules
-
-mozilla-detekt-rules:
-  active: true
-  MozillaBannedPropertyAccess:
-    active: true
-    # BuildConfig.Debug: This property tests whether the application was built
-    # with the debuggable flag or not. Use a check for different build variants,
-    # instead.
-    bannedProperties: 'BuildConfig.DEBUG'
-  MozillaStrictModeSuppression:
-    active: true
-  MozillaCorrectUnitTestRunner:
-    active: true
-  MozillaRunBlockingCheck:
-    active: true
-  MozillaUseLazyMonitored:
-    active: true
```

Comment:
This is unique to Fenix, but I don't think we want to remove these custom rules

Reply:
Gah, copy & paste fail on my part! Fixed.